js/src/forum/components/PostMeta.js

```diff
@@ -29,9 +30,9 @@ export default class PostMeta extends Component {
 
     return (
       <div className="Dropdown PostMeta">
-        <a className="Dropdown-toggle" onclick={selectPermalink} data-toggle="dropdown">
+        <Button className="Button Button--reset Dropdown-toggle" onclick={selectPermalink} data-toggle="dropdown">
```

Comment:
the timestamp looks colored at all times, can we instead use `Button--text` here and just add a `vertical-align: bottom;` to `.Button--text` ? it's a lot more consistent.